src/adm/filter.rs

```diff
@@ -381,34 +380,24 @@ impl AdmFilter {
         device_info: &DeviceInfo,
         tags: &mut Tags,
         metrics: &Metrics,
-    ) -> Option<Tile> {
+    ) -> HandlerResult<Option<Tile>> {
```

Comment:
This might not need to return a Result any longer (I'm guessing it was added for NoTilesForCountry)?